### ISSUE TITLE
AutAutronicIcon does not have a property matching the name 'aria-hidden'

### ISSUE THREAD

**Issue by hakenr** [havit/Havit.Blazor#1065]
Runtime error after upgrading to 4.8.0:

![Image](https://github.com/user-attachments/assets/13dfa979-01e1-404e-93a0-8137a3f3a58b)

**Comment by hakenr** [comment]
`AutAutronicIcon` does not implement attribute splatting.
Double-checked the documentation - it is documented as a requirement for icon rendering component.
